topology/University of Connecticut/UConn-OSG/UConn-OSG.yaml

```diff
@@ -67,7 +71,7 @@ Resources:
           uri_override: gluskap.phys.uconn.edu:2811
     VOOwnership:
       Gluex: 100
-  UConn-OSG_SE:
+  GlueX_submit_host:
```

Comment:
Just a note for `GlueX_submit_host` - resource names need to be globally unique, so if there might other GlueX submit nodes outside of UConn-OSG, you may want to consider including UConn-OSG in the name somehow.

Reply:
Ok, I thought the resource namespace is scoped by the site and resource group names. Wasn't that implied by the slash construction in the full resource path? Can I retract this pull request and try again?

Reply:
Unfortunately both the resource group and resource group names must be unique across the entire OSG topology, despite them appearing at different parts of the hierarchy.

Yes feel free to make a new PR.